specs/scoring_spec.rb

```diff
@@ -42,21 +41,32 @@
 
   describe 'highest_score_from' do
     it 'returns nil if no words were passed' do
+      Scrabble::Scoring.highest_score_from([]).must_be_nil
     end
 
     it 'returns the only word in a length-1 array' do
+      Scrabble::Scoring.highest_score_from(['dog']).must_equal "dog"
     end
 
     it 'returns the highest word if there are two words' do
-    end
+      array_of_words = ["dog", "fish"]
+      Scrabble::Scoring.highest_score_from(array_of_words).must_equal "fish"
```

Comment:
Like we discussed in class, it would probably be wise to check multiple orderings of these words, that is, both `['dog', 'fish']` and `['fish', 'dog']`. You'd be surprised how often this sort of thing turns up a bug.